parse/test.py

```diff
@@ -1,63 +1,65 @@
 import requests
 from bs4 import BeautifulSoup
 import regex as re
+from selenium import webdriver
+import csv
+from datetime import datetime, timedelta
+
+
+flats = []
+URL = 'https://r.onliner.by/ak/'
+chromedriver = r'..\resources\chromedriver.exe'
 
 
-def parse_storgame():
-    URL = 'https://stopgame.ru/topgames'
-    r = requests.get(URL)
-    html = BeautifulSoup(r.content, 'html.parser')
-    for elem in html.select('.game-summary-horiz'):
-        title = elem.select('.caption > a')
-        print(title[0].text)
-
-
-def parse_onliner():
-    URL = 'https://r.onliner.by/ak/?rent_type%5B%5D=room&only_owner=true#bounds%5Blb%5D%5Blat%5D=53.89129696150157&bounds%5Blb%5D%5Blong%5D=27.304834215169524&bounds%5Brt%5D%5Blat%5D=53.960988463831015&bounds%5Brt%5D%5Blong%5D=27.75315394037773'
-    HEADERS = {
-        'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.83 Safari/537.36',
-        'params': '*/*'}
-    r = requests.get(URL, headers=HEADERS, params=None)
-
-    html = BeautifulSoup(r.content, 'html.parser')
-    print(html)
-    # for elem in html.select('.classified > a'):
-    #     print(elem)
-    # for elem in html.select('.classified'):
-    #     print(elem)
-    items = html.find_all('a', class_='classified')
-    # price = elem.select('.classified__price classified__price_secondary')
-    # time = elem.select('.classified__time > a')
-    # adress = elem.select('.classified__caption-item classified__caption-item_adress')
-    # print(time.text, price, adress)
-
-
-def selenium():
-    from selenium import webdriver
-    flats = []
-    URL = 'https://r.onliner.by/ak/'
-    chromedriver = 'resources\chromedriver.exe'
-    # def open_browser(chromedriver):
+def parse_date(date_time, time=None):
+    data_time = {
+        'секунду,секунды,секунд': 1,
+        'минуту,минуты,минут': 60,
+        'час,часов,часа': 3600,
+        'день,дня,дней': 86400,
+        'неделя,недели,недель': 604800,
+    }
```

Comment:
It has to be the constant. You are create the dictionary whenever you call the 'parse_date' function